src/components/templates/Tooltip/Tooltip.tsx

```diff
@@ -51,7 +53,15 @@ export const Tooltip = ({
                             role='tooltip'
                             static
                         >
-                            <div className='typography-caption-3 relative w-fit overflow-hidden whitespace-normal rounded-lg border border-black-20 bg-gunMetal p-4 text-left text-neutral-8 shadow-dropdown'>
+                            <div
+                                className={clsx(
+                                    'typography-caption-3 relative w-fit overflow-hidden whitespace-normal rounded-lg border border-black-20 bg-gunMetal p-4 text-left text-neutral-8 shadow-dropdown',
+                                    {
+                                        'border-none bg-yellow-900 text-neutral-50':
+                                            severity === 'warning',
```

Comment:
Shall we add styling for `severity === 'info'` here? And default styling will be applied via default `severity` value on line 11.

Reply:
Can we try

`{'border-black-20 bg-gunMetal text-neutral-8': severity === 'info'}` here so we can see `info`-specific styling